### ISSUE TITLE
Is my proposal relevant web-app-launch?

### ISSUE THREAD

**Issue by hanguokai** [WICG/web-app-launch#86]
Hi,

I have a new proposal for `registerProtocolHandler()` and service worker. You can see it here: https://github.com/whatwg/html/issues/8596 and https://github.com/w3c/ServiceWorker/issues/1665

I want to ask: is this proposal related to web-app-launch?

Thanks

**Comment by alancutter** [comment]
Yes it is, you most likely want `"launch_handler": { "client_mode": "service-worker" }`: https://github.com/WICG/web-app-launch/blob/main/launch_handler.md#possible-extensions-to-this-proposal
The reason for doing it at this level is so it can affect all launches, not just protocol handler launches.
I recommend not altering the existing `registerProtocolHandler()` API and instead pursuing this API.
Thanks for checking in!
